modules/tinymce/src/plugins/lists/main/ts/core/ListNumbering.ts

```diff
@@ -0,0 +1,100 @@
+import { Arr, Optional, Strings } from '@ephox/katamari';
+
+export interface ListDetail {
+  start: string;
+  listStyleType: string | false;
+}
+
+const enum ListType {
+  UpperAlpha,
+  LowerAlpha,
+  Numeric,
+  Unknown
+}
+
+// Example: 'AB' -> 28
+const parseAlphabeticBase26 = (str: string): number => {
+  const chars = Arr.reverse(Strings.trim(str).split(''));
+  const values = Arr.map(chars, (char, i) => {
+    const charValue = char.toUpperCase().charCodeAt(0) - 'A'.charCodeAt(0) + 1;
+    return Math.pow(26, i) * charValue;
+  });
+  return Arr.foldl(values, (sum, v) => sum + v, 0);
+};
+
+// Example: 28 -> 'AB'
+const composeAlphabeticBase26 = (value: number): string => {
+  value--;
+  if (value < 0) {
+    return '';
+  } else {
+    const remainder = value % 26;
+    const quotient = Math.floor(value / 26);
+
+    const rest = composeAlphabeticBase26(quotient);
+    const char = String.fromCharCode('A'.charCodeAt(0) + remainder);
+
+    return rest + char;
+  }
+};
```

Comment:
This is not far off how we did it in our previous editor that supported updating list start values! I'll note it in JIRA since it's a private repo link.

The code might be worth bookmarking if we later want to support start numbering for roman lists as well.